tests/utils/testing_utils_common.py

```diff
@@ -0,0 +1,22 @@
+import tempfile
+import torch
+
+class BaseModelTester:
+    all_model_names = None
+    trl_model_class = None
+
+    def test_from_save(self):
```

Comment:
another useful test would be if we can load a LM with a v head, save it and then load it as a pure LM. that's what most people will likely use for inference.

Reply:
Great catch! 
Added a test `test_from_save_transformers`, you can now train a model with `trl` save it, and load it back with `transformers`, ins't that cool? :D